cellpack/autopack/interface_objects/representations.py

```diff
@@ -119,7 +119,7 @@ def get_mesh_path(self):
         else:
             if self.mesh["path"] == "default":
                 return f"{self.DATABASE}/geometries/{self.mesh['name']}"
-            return f"{self.mesh['path']}{self.mesh['name']}"
+            return f"{self.mesh['path']}/{self.mesh['name']}"
```

Comment:
so we don't get concatenated paths

Reply:
Could be worth looking into using `pathlib.Path` for paths in general. They handle all the corner cases of concatenation and also help when using windows style paths. 

> from pathlib import Path
> Path('one/path') / 'further/paths'